apis/fluentbit/v1alpha2/plugins/output/open_search_types.go

```diff
@@ -0,0 +1,218 @@
+	// Optional username credential for access
+	HTTPUser *plugins.Secret `json:"httpUser,omitempty"`
+	// Password for user defined in HTTP_User
+	HTTPPasswd *plugins.Secret `json:"httpPassword,omitempty"`
+	// Index name
+	Index string `json:"index,omitempty"`
+	// Type name
+	Type string `json:"type,omitempty"`
+	// Enable Logstash format compatibility.
+	// This option takes a boolean value: True/False, On/Off
+	LogstashFormat *bool `json:"logstashFormat,omitempty"`
+	// When Logstash_Format is enabled, the Index name is composed using a prefix and the date,
+	// e.g: If Logstash_Prefix is equals to 'mydata' your index will become 'mydata-YYYY.MM.DD'.
+	// The last string appended belongs to the date when the data is being generated.
+	LogstashPrefix string `json:"logstashPrefix,omitempty"`
+	// Time format (based on strftime) to generate the second part of the Index name.
+	LogstashDateFormat string `json:"logstashDateFormat,omitempty"`
+	// When Logstash_Format is enabled, each record will get a new timestamp field.
+	// The Time_Key property defines the name of that field.
+	TimeKey string `json:"timeKey,omitempty"`
+	// When Logstash_Format is enabled, this property defines the format of the timestamp.
+	TimeKeyFormat string `json:"timeKeyFormat,omitempty"`
+	// When Logstash_Format is enabled, enabling this property sends nanosecond precision timestamps.
+	TimeKeyNanos *bool `json:"timeKeyNanos,omitempty"`
+	// When enabled, it append the Tag name to the record.
+	IncludeTagKey *bool `json:"includeTagKey,omitempty"`
+	// When Include_Tag_Key is enabled, this property defines the key name for the tag.
+	TagKey string `json:"tagKey,omitempty"`
+	// When enabled, generate _id for outgoing records.
+	// This prevents duplicate records when retrying OpenSearch.
+	GenerateID *bool `json:"generateID,omitempty"`
+	// If set, _id will be the value of the key from incoming record and Generate_ID option is ignored.
+	IdKey string `json:"idKey,omitempty"`
+	// Operation to use to write in bulk requests.
+	WriteOperation string `json:"writeOperation,omitempty"`
+	// When enabled, replace field name dots with underscore, required by Elasticsearch 2.0-2.3.
+	ReplaceDots *bool `json:"replaceDots,omitempty"`
+	// When enabled print the elasticsearch API calls to stdout (for diag only)
+	TraceOutput *bool `json:"traceOutput,omitempty"`
+	// When enabled print the elasticsearch API calls to stdout when elasticsearch returns an error
+	TraceError *bool `json:"traceError,omitempty"`
+	// Use current time for index generation instead of message record
+	CurrentTimeIndex *bool `json:"currentTimeIndex,omitempty"`
+	// Prefix keys with this string
+	LogstashPrefixKey string `json:"logstashPrefixKey,omitempty"`
+	// When enabled, mapping types is removed and Type option is ignored. Types are deprecated in APIs in v7.0. This options is for v7.0 or later.
+	SuppressTypeName string `json:"suppressTypeName,omitempty"`
```

Comment:
https://docs.fluentbit.io/manual/pipeline/outputs/opensearch
`When enabled, mapping types is removed and Type option is ignored.`
This appears to have only `on` and `off` values, which seems to be possible with bool types?

Reply:
Thanks, I would to fix now.

Reply:
Done.